easybuild/easyconfigs/b/BioPerl/BioPerl-1.7.2-foss-2019a-Perl-5.28.1.eb

```diff
@@ -0,0 +1,30 @@
+# easybuild easyconfig
+#
+#  John Dey jfdey@fredhutch.org
+#
+#  Fred Hutchinson Cancer Research Center
+
+easyblock = 'PerlModule'
+
+name = 'BioPerl'
+version = '1.7.2'
+versionsuffix = '-Perl-%(perlver)s'
```

Comment:
I don't think we need the versionsuffix for Perl. There is just 1 Perl install per toolchain. Could any else confirm this please?

Reply:
All packages providing modules for Perl define a versionsuffix. I would not arbitrarily change that for a single package without having some kind of announcement regarding the (non-written?) rules on name scheme for these packages.

Reply:
Well, the reason they have is because
1. There used to be multiple versions  of libraries within a toolchain. This wasn't worth all the problems it caused, so this is heavily discouraged nowdays.
2. Someone added a new a new config, they look at all the others, seeing a suffix, and imitate. 

So, there has been an effort of dropping version-suffixes in new configs when possible (though, some slip through). I.M.H.O. there is no reason to have the perl suffix around anymore, but I welcome others opinion

Reply:
> So, there has been an effort of dropping version-suffixes in new configs when possible (though, some slip through).

That is not true in this case. All packages currently in the tree that provide modules for Perl define a versionsuffix, all of them. So what you are proposing would break the consistent naming of those packages  and I love consistency.
Regardless of the usefulness (or not) of a versionsuffix, I think that it is a bad strategy to request the drop of versionsuffix on individual PRs as they come in. At the end of the day those are gonna be chosen randomly by the chance of catching the attention of a particular mod or not. Such a change needs a proper public announcement, so all people contributing is aware of it. Otherwise, we will end up with a mix of packages with and without versionsuffix, which is useless and confusing.

Reply:
There will never be retroactive changes to configs that has been part of a EB release (unless absolutely necessary), as it would break things. Such changes in EB are always only _going forward_. And there actually are just 3 modules that has a Perl-5.28.1 suffix, of which 2 of them are so recent that they have not been part of a EB release yet, so, technically could be easily addressed if we wanted to. 

git provides some perl components, no suffix. intltool used to have the suffix, was removed for 8.2.0. 
I'm just for consistency with what we do for Python, Java, and every other dependency nowdays (2019+). If the other maintainers disagree with me here, just dismiss this.

Reply:
You are both right. On one side we are working in order to have a single version of a given interpreter/library per toolchain, in order to reduce the possible permutations and consequently the need for `versionsuffix`. On the other side, although we started removing version suffixes, we have set rules only for the easyconfig packages that depends on `Python`.
I would recommend sticking with the `versionsuffix` until we add Travis tests that will enforce or not the need for it.